plugin/src/com/microsoft/alm/plugin/idea/tfvc/core/TFSChangeProvider.java

```diff
@@ -104,16 +101,10 @@ public void getChanges(@NotNull final VcsDirtyScope dirtyScope,
 
         List<PendingChange> changes;
         try {
-            // Status command requires credentials (even if it doesn't use them in most cases). On Windows, it is able
-            // to discover current user's NTLM credentials and use them (even if they don't usually match the
-            // credentials used to access the TFS, it doesn't matter). For other OSs, where TF client couldn't get any
-            // default credentials, we'll obtain the valid credentials from the current server context.
-            ServerContext serverContext = SystemInfo.isWindows ? null : myVcs.getServerContext(false);
-            changes = EULADialog.executeWithGuard(
-                    project,
-                    () -> CommandUtils.getStatusForFiles(project, serverContext, pathsToProcess));
+            ServerContext serverContext = myVcs.getServerContext(true);
+            changes = TfvcClient.getInstance(project).getStatusForFiles(serverContext, pathsToProcess);
         } catch (final Throwable t) {
-            logger.warn("Failed to get changes from command line. roots=" + StringUtils.join(pathsToProcess, ", "), t);
+            logger.error("Failed to get changes from command line. roots=" + StringUtils.join(pathsToProcess, ", "), t);
```

Comment:
Actually, I think it should've been an error from the very beginning :)

But, here's an explanation why I think we should change it now: the new client is highly experimental, and _probably_ is more error-prone than an old client. On the other hand, this particular functionality is critical (because it is the first operation that's been ported to the new client).

I don't want the new client's failures to go silent and unnoticed, that's why I think we should change the log level here in this PR.